Vagrantfile

```diff
@@ -30,14 +32,14 @@ end
 @chefip = @network[:chef][:ip]
 
 # Network sanity checks.
-fail 'Non-unique ips' if @network.collect { |_, v| v[:ip] }.uniq!
+raise 'Non-unique ips' if @network.collect { |_, v| v[:ip] }.uniq!
 
-fail 'Non-unique hostnames' if @network.collect { |_, v| v[:hostname] }.uniq!
+raise 'Non-unique hostnames' if @network.collect { |_, v| v[:hostname] }.uniq!
 
-fail 'Non-unique ports' if @network.collect { |_, v| v[:ports].keys }.flat_map { |v| v }.uniq!
+raise 'Non-unique ports' if @network.collect { |_, v| v[:ports].keys }.flat_map { |v| v }.uniq!
 
 def default_omnibus(config)
-  config.omnibus.chef_version = '12'
+  config.omnibus.chef_version = '12.19.36'
```

Comment:
But we're pegging to 12.18.31 in the Gemfile ?

Reply:
I must have been testing upgrading the chef version but forgot to update the gemfile

Reply:
d744f3347c912df0f7e202ef85b2f8935a7b7e07